### PR TITLE
Added open source doc

### PR DESCRIPTION
**Pull request by Lloyd-Jackman-UKPL**
Per conversation with @riccardoferretti I am creating this PR to add a doc on open source and privacy concerned alternatives to the standard VS Code and Github workflows.

### CODE REVIEW

File: docs/Achieving Greater Privacy and Security.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The change you will notice in using VS Code versus VS Codium - simply speaking, none. It is, in just about every way you will think of, the same IDE, just without the Microsoft proprietary licence and telemetry.  Your Foam experience will remain as smooth and productive as before the change.
+
+## Version Control and Replication
+
+In Foam's [Getting Started](https://foambubble.github.io/foam/#getting-started) section, the set up describes how to set up your notes with a GitHub repository in using the template provided.  Doing so provides the user with both the ability to see commits made and therefore versions of their notes, but also allows the user to work across devices or collaborate effectively with other users.  These are all strong use cases, but the web-hosting of the content and closed-source nature of GitHub, being owned by Microsoft, may present some users with issues.
```
Comment (riccardoferretti) [inline docs/Achieving Greater Privacy and Security.md:15]:
```suggestion
In Foam's [Getting Started](https://foambubble.github.io/foam/#getting-started) section, the set up describes how to set up your notes with a GitHub repository in using the template provided.  Doing so provides the user with the ability to see commits made and therefore versions of their notes, allows the user to work across devices or collaborate effectively with other users, and makes publishing to GitHub pages easy.
It's important at the same time to point out the closed-source nature of GitHub, being owned by Microsoft.
```

File: docs/Achieving Greater Privacy and Security.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+One alternative approach could be to use GitLab, an open source alternative to GitHub.  Whilst it certainly effectively deals with the issue of transparency, it does also collect usage details and sends your content across the internet.
```
Comment (riccardoferretti) [inline docs/Achieving Greater Privacy and Security.md:17]:
```suggestion
One alternative approach could be to use [GitLab](https://gitlab.com/), an open source alternative to GitHub.  Whilst it improves on the aspect of transparency, it does also collect usage details and sends your content across the internet.  
And of course data is still stored in clear in the cloud, making it susceptible to hacks of the service.
```

File: docs/Achieving Greater Privacy and Security.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## A Debt?
+
+Foam development has relied upon it being built on VS Code with its website and documentation hosted on GitHub.  Despite Foam itself being open source, it would be easy to feel that swapping out such items is ungrateful for Microsoft's (passive) contribution.  It would, however, be equally poor to not accept the varying use cases and requirements of Foam users and reflect how these default choices may not be suitable for all.  In exploring the above, we maintain Foam's place in a broad set of applications with a multitude of interfaces.
```
Comment (riccardoferretti) [inline docs/Achieving Greater Privacy and Security.md:25]:
```suggestion
```

**Review by riccardoferretti — COMMENTED** [review]
Thanks for drafting this page!
I have proposed a few changes, and stylistically I would make it easier for people to read the document by having shorter sentences that go straight to the point (consider that english might not be the first language for many Foam users).

Thoughts?

**Comment by Lloyd-Jackman-UKPL** [comment]
@riccardoferretti thanks for the feedback. The best writing is rewriting 😊

**Review by riccardoferretti — APPROVED** [review]

**Comment by riccardoferretti** [comment]
@allcontributors add @Lloyd-Jackman-UKPL for docs

**Comment by riccardoferretti** [comment]
@allcontributors add @Lloyd-Jackman-UKPL for docs

**Comment by Lloyd-Jackman-UKPL** [comment]
> @riccardoferretti
> 
> Could not find the user `Lloyd-Jackman#/#UKPL` on github.

Please add me then 😊

**Pull request merged**